.github/workflows/build.yaml

```
@@ -0,0 +1,27 @@
+name: Ubuntu Build Test
+
+on:
+    pull_request: {}
+    push: {}
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.16
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Build Test
+      run: |
+        make -C lib/go protoc
```

```suggestion
        make
        make -C lib/go protoc
```
^ `make` as a separate step, before installing deps, to run tests. which works as expected if you also ...
1. update Makefile and lib/go/Makefile: s/TRAVIS/GITHUB_ACTIONS/
2. drop the .travis.yml file

---

fixed the comments, PTAL.